Translated using Weblate (Russian) for 1.21

marisathewitch opened the pull request:
Currently translated at 99.7% (5521 of 5534 strings)

Commit:
Translated using Weblate (Russian)

Currently translated at 99.7% (5521 of 5534 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://kardini.tplinkdns.com/projects/gregtech-ceu/gregtech-ceu-modern-1-20-1/ru/

screret reviewed (changes requested):
Please also remove the unused old keys.

marisathewitch committed:
Merge branch 'GregTechCEu:1.21' into 1.21

weblate committed:
Translated using Weblate (Russian)

Currently translated at 99.7% (5521 of 5534 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.21)
Translate-URL: http://kardini.tplinkdns.com/projects/gregtech-ceu/gregtech-ceu-modern-1-21/ru/

weblate committed:
Update translation files

Updated by "Cleanup translation files" add-on in Weblate.

Translation: Gregtech: CEu/GregTech CEu Modern (1.21)
Translate-URL: http://kardini.tplinkdns.com/projects/gregtech-ceu/gregtech-ceu-modern-1-21/

marisathewitch commented:
> Please also remove the unused old keys.

Done
